exampleCourse/questions/template/multiple-choice/none-of-the-above/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+  <p>Which of the following is <strong>not</strong> a common sorting algorithm taught in introductory Computer Science courses?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="sorting">
+  <pl-answer correct="false">Bubble Sort</pl-answer>
+  <pl-answer correct="false">Merge Sort</pl-answer>
+  <pl-answer correct="false">Quick Sort</pl-answer>
+  <pl-answer correct="true">None of the above</pl-answer>
```

Comment:
Don't explicitly add this answer choice, use the elements built in none of the above option instead: https://prairielearn.readthedocs.io/en/latest/elements/#customizations_9

Reply:
Done https://github.com/PrairieLearn/PrairieLearn/pull/12139/commits/bee5004c81e217755cadcb61821870526805bd9f